src/style.css

```diff
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 > .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--5 > .stars__star {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 > .stars__star:nth-child(n+1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 > .stars__star:nth-child(n+2) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--2 > .stars__star:nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 > .stars__star:nth-child(n+3) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--3 > .stars__star:nth-child(-n + 3) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
combine all selectors with background-image with active stars

Reply:
I've fixed problem, can you check?